combat-trainer.lic

```diff
@@ -970,7 +971,7 @@ class SpellProcess
     release_cyclics if data['cyclic']
     command = 'pre'
     command = data['skill'] != 'Targeted Magic' ? 'pre' : 'tar' if data['skill']
```

Comment:
Sorry, can you be more specific? Should 973 pull the prep command from spell_data?